trezorlib/tx_api.py

```diff
@@ -97,6 +112,13 @@ def get_tx(self, txhash):
             o = t._add_bin_outputs()
             o.amount = int(Decimal(vout['value']) * 100000000)
             o.script_pubkey = binascii.unhexlify(vout['scriptPubKey']['hex'])
+            if(self.bip115 and len(o.script_pubkey) == 63 and o.script_pubkey[-1:] == b'\xb4'):
+                o.block_hash = o.script_pubkey[-37:-5]
+                o.block_height = int.from_bytes(o.script_pubkey[-4:-1], byteorder='little')
+
+            else:
+                o.block_hash = None
+                o.block_height = None
```

Comment:
this `else` here is a no-op (all fields are preinitialized to `None` anyway) so please remove it